screens/Scan/ScanLayout.tsx

```diff
@@ -43,8 +47,42 @@ export const ScanLayout: React.FC = () => {
 
   return (
     <React.Fragment>
+      <VerifyIdentityOverlay
+        isVisible={
+          controller.isVerifyingIdentity &&
+          controller.flowType === FlowType.MINI_VIEW_SHARE_WITH_SELFIE
```

Comment:
flow type check is not necessary as we will not reach `VerifyingIdentity` state in normal share flow